tests/E2E/test/campaigns/high/13_employee/1_employee.js

```diff
@@ -24,18 +25,25 @@ scenario('Create employee', client => {
   test('should set "Password" input', () => client.waitAndSetValue(Employee.password_input, '123456789'));
   test('should choose "Permission profile" option', () => client.waitAndSelectByValue(Employee.profile_select, '4'));
   test('should click on "Save" button', () => client.waitForExistAndClick(Employee.save_button));
+  test('should verify the appearance of the green validation', () => client.checkTextValue(Employee.success_panel, 'Successful creation.'));
 }, 'common_client');
 
 scenario('Check the employee creation', client => {
   test('should search the created employee', () => client.waitAndSetValue(Employee.email_search_input, 'demo' + date_time + '@prestashop.com'));
   test('should click on "Search" button', () => client.waitForExistAndClick(Employee.search_button_team));
-  test('should check the result', () => client.checkTextValue(Employee.search_result, "1"));
+  test('should check the result', () => {
+    return promise
+      .then(() => client.waitForVisible(Employee.search_result))
+      .then(() => client.getTextInVar(Employee.search_result,'employee_number'))
+      .then(() => tab['employee_number'] = tab['employee_number'].split('(')[1].split(')')[0])
```

Comment:
use regex instead please

Reply:
What do you think about this ?
`tab['employee_number'] = /\(([^)]+)\)/.exec(tab['employee_number'])[1]`

Reply:
@PierreRambaud Do you except changes here?

Reply:
Yep, he already did it here https://github.com/PrestaShop/PrestaShop/pull/13545/commits/a21134a38ca1b4adfd481bf61a32a7e975e68996

Reply:
Updated in this one [26affc7](https://github.com/PrestaShop/PrestaShop/pull/13545/commits/26affc7d522e41227b48f90383664749526e6424)

Reply:
Not here

Reply:
done here [`dcf3b30`](https://github.com/PrestaShop/PrestaShop/pull/13545/commits/dcf3b301ec7d755b82141cba91fb2b0054ff2b89)